NetCoreRestApi/DataLayer/UnitOfWorks/IUnitOfWork.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Linq;
+
+namespace DataLayer.UnitOfWorks
+{
+    public interface IUnitOfWork<T>
+    {
+        T GetById(int id);
+        IQueryable<T> GetAll();
+        IQueryable<T> Create(T entity);
```

Comment:
Use ICollection<T> entities instead of T entity

Reply:
done